storage/connectors/jdbc/src/main/java/feast/storage/connectors/jdbc/retriever/AbstractJdbcQueryTemplater.java

```diff
@@ -254,7 +254,8 @@ protected String createTimestampLimitQuery(String leftTableName) {
       for (String entity : featureSetQueryInfo.getEntities()) {
         if (!entities.contains(entity)) {
           entities.add(entity);
-          entityColumns.add(String.format("%s.%s", table, entity));
+          // parse entities from FEATURE variant column
+          entityColumns.add(String.format("%s.feature:%s AS %s", table, entity, entity));
```

Comment:
is this specific to snowflake?

Reply:
good point. let me double check

Reply:
@dr3s @jmelinav I have a question regarding the Postgres implementation. In this PR, shall I also update the Postgres templater? Or, we don't support Postgres for now?

Reply:
@zhangchi1 for now you can comment the postgres implementation and make sure to move snowflake specific code under snowflake classes.